src/savers/tvg/tvgTvgSaver.cpp

```diff
@@ -203,31 +203,28 @@ TvgBinCounter TvgSaver::serializeFill(const Fill* fill, TvgBinTag tag)
 }
 
 
-TvgBinCounter TvgSaver::serializeStroke(const Shape* shape)
+TvgBinCounter TvgSaver::serializeStroke(const Shape* shape, const Fill* fill, uint8_t color[4])
 {
```

Comment:
I didn't want to call `serializeStroke` if it not necessary.
In the svg the default value of the stroke width is 1, so we need to check the color alpha and the fill (l.287 and l.288)
so to prevent calling the same functions twice, I passed the fill and color values as paramters.

Reply:
@mgrudzinska i don't much like this because it looks more complex... in the end, it's costless.

Reply:
@hermet please check now. I call `strokeColor` and `strokeFill` before calling `serializeStroke` since otherwise for each shape from the svg file a stroke will be serialized, even if it's not visible.